### PR TITLE
Created scaffolding script

### PR DESCRIPTION
**Pull request by mschoettner**
Added a file called script-scaffolding.py that can be used as a
basis for each classifier that we want to train. How training and
test set are to be defined as well as the definition of the model
have been left empty.
@anproulx @emilyemchen @guenounz Please check for errors

### CODE REVIEW

**Review by anproulx — COMMENTED** [review]
Super!!! I have one small comment about the part where you extract the target vector... it is not necessary to iterate to extract the data since sklearn has been conceived to work with pandas dataframes. to extract the Y variable you can only do : Y=phenotypic["DX_GROUP"] and you will be able to use that in your model after. I can make the changes if you'd like

**Review by anproulx — APPROVED** [review]

**Review by emilyemchen — COMMENTED** [review]
Thanks so much for this! A few questions that I'm not sure about so I won't request changes: 

1. If we're not plotting we don't need `%matplotlib inline` right? I have that in my script but maybe it isn't necessary
2. You did say subfolder `ABIDE_pcp` but maybe we specify that the super folder is called `nilearn_data`?

**Review by anproulx — COMMENTED** [review]
I will later today add the code to remove half the matrix, and I will apply PCA so that we reduce the amount features used in our prediction models (while stile keeping 99% of explained variance)

**Comment by mschoettner** [comment]
@emilyemchen 

1. Yes, exactly, `%matplotlib inline` makes plots appear below cells if you run them. In a regular Python script they do not serve any purpose.
2. Could be, I think I renamed the super folder (:

**Comment by mschoettner** [comment]
> 
> 
> I will later today add the code to remove half the matrix, and I will apply PCA so that we reduce the amount features used in our prediction models (while stile keeping 99% of explained variance)

@anproulx that sounds awesome! Maybe open an issue for that for the sake of transparency :)
